lib/src/test/java/vmath/Mat3Test.java

```diff
@@ -120,4 +120,20 @@ void givenArrayWithNaNs_whenMat3Created_thenThrowsException() {
         };
         assertThrows(ArithmeticException.class, () -> new Mat3(a2));
     }
+
+    @Test
+    void givenVec3_whenMultiplied_thenResultIsCorrect() {
+        Mat3 m = new Mat3(
+                1.0f, 2.0f, 3.0f,
+                4.0f, 5.0f, 6.0f,
+                7.0f, 8.0f, 9.0f
+        );
```

Comment:
Please use 4 spaces to indent this section, like the rest.

Reply:
Please review  the updated file, I have fixed the indentation for `mul()`  test. Thank you